.gitignore

```diff
@@ -4,4 +4,5 @@ coverage
 /public
 !/public/index.html
 !/public/fortmaticG6oWpqbFTZ3fc3g2.html
-/public-clay
\ No newline at end of file
+/public-clay
+.DS_Store
```

Comment:
Can you add this to your local settings rather than having env-specific settings in the repo please?

Reply:
Still seems to be here with latest changes?

Reply:
I forgot to hit git push, i've just removed it now.